### ISSUE TITLE
WCAG: Pending button contrast

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2199]
The button with text "Pending" has a pink background # EEF2F3 and white text. The color contrast is 1.1: 1. Normally, the buttons that are disabled do not have to meet the contrast requirements. Here this button shows an important message that the request is pending. This information must have sufficient color contrast. See screenshot 4.

A visually impaired user can miss this text and as a result not know the status of his request.

Provide a minimum contrast of 4.5: 1.

## Solution: Text change color to Black in this type of buttons

![Picture4](https://user-images.githubusercontent.com/33207036/111763581-9cb32e00-88a2-11eb-9e80-49c806b715a8.png)
